src/components/MeetInfo-Table/index.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { useTable } from "react-table";
+import { TableWrapper } from "./MeetInfoTable.style";
+
+const Table = ({ columns, data }) => {
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        rows,
+        prepareRow,
+    } = useTable({
+        columns,
+        data,
+    });
+
+    return (
+        <TableWrapper>
+            <h1 className="meetings-table-title">Layer5 Meetings</h1>
+            <h3 className="meetings-table-subtitle">Everyone is welcome to join. Engage!</h3>
+            <table {...getTableProps()}>
+                <thead>
+                    {headerGroups.map(headerGroup => (
+                        <tr key={"table-header"} {...headerGroup.getHeaderGroupProps()}>
+                            {headerGroup.headers.map((column) => (
+                                <th key={column} {...column.getHeaderProps()}>{column.render("Header")}</th>
+                            ))}
+                        </tr>
+                    ))}
+                </thead>
+                <tbody {...getTableBodyProps()}>
+                    {rows.map((row, i) => {
+                        prepareRow(row);
+                        return (
+                            <tr key={`row${i}`} {...row.getRowProps()}>
+                                {row.cells.map(cell => {
+                                    if( cell["column"]["id"] === "meeting" ){
```

Comment:
Instead of doing if-else again and again go can simply change the accessor function. Check #1346 for reference